apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/CreateModelExtractJobOptions.cs

```diff
@@ -27,12 +27,27 @@ public sealed class CreateModelExtractJobOptions : JobCreationOptions
         /// </summary>
         public ModelFormat? DestinationFormat { get; set; }
 
+        /// <summary>
+        /// Optional action to perform after preparing the request. If this property is non-null,
+        /// the <see cref="JobConfigurationExtract"/> used for a request will be passed to the delegate
+        /// before the request is executed. This allows for fine-grained modifications which aren't
+        /// otherwise directly supported by the properties in this options type.
+        /// </summary>
+        /// <remarks>
+        /// Prefer the properties on this type over this modifier to prepare the request.
+        /// Only use this modifier to configure aspects for which there are no properties available.
+        /// This modifier is applied to the request after all properties on this type have been applied.
+        /// The delegate is only called once per operation, even if the request is automatically retried.
+        /// </remarks>
+        public System.Action<JobConfigurationExtract> ConfigurationModifier { get; set; }
```

Comment:
Any reason for not adding a using directive for the `System` namespace? We rarely use fully-qualified names unless we really have to.